test/mockRaf.js

```diff
@@ -0,0 +1,35 @@
+export default {
```

Comment:
Return factory and create object with empty values `beforeEach`, so you will avoid potential value clashes and your tests will be completely independent.